docs/execution-guide.md

```diff
@@ -32,43 +31,78 @@ PID|1|7777555^4^M11^test^MR^University Hospital^19241011^19241012|PATID7755^5^M1
-token=$(az account get-access-token --resource=https://phdi-dev-fhir-server.azurehealthcareapis.com --query accessToken --output tsv)```.
-Replace the URL after ```--resource=``` with the URL you copied above. Hit enter to run this command.
-
-26. Then, type in this command: ```RESPONSE=$(curl -X GET --header "Authorization: Bearer $token" https://phdi-dev-fhir-server.azurehealthcareapis.com/Patient?family=DOE&given=JOHN)```
-Replace the URL in-between ```Bearer $token``` and ```/Patient``` with the URL you copied above. Hit enter to run this command.
-
-
-27. Finally, press the up arrow until you see this command: ```echo $RESPONSE | jq``` Hit enter to run this command.
+If you would like, feel free to confirm that this is the case by inspecting the file directly in the text editor of your choice.
+
+### Overview
+This how-to guide is divided into 3 sections:
+1. Accessing your Azure account
+2. Running data through the pipeline
+3. Viewing data in the FHIR server
+
+### Access your Azure account 
+> **Note**: If this is your first time running data through the pipeline, we recommend having this guide and the Azure portal open side-by-side.  
+
+1. Open [https://portal.azure.com/](https://portal.azure.com/) in your browser and log in with your username and password.![azure-portal](./images/azure-portal.png)
+1. Make sure you're logged into the account that has access to the Azure resource group you have used so far. To check, go to the "Azure services" heading and click on `Resource groups`.![azure-portal-resource-groups](./images/azure-portal-resource-groups.png) 
+1. Click into the appropriate Azure resource group.![azure-portal-resource-group](./images/azure-portal-resource-group.png) 
+
+### Upload and run data through the pipeline
+> **Note**: If you prefer, you can upload data using the [Azure Storage Explorer Tool](https://azure.microsoft.com/en-us/products/storage/storage-explorer/). We don't provide instructions for using that tool here, but the broad strokes will be the same - you'll need to upload `sample-data/VXU-V04-01_success_single.hl7` to the `source-data` container in your PHI storage account.
+
+1. Within your `Resource group`, filter down to view only `Storage account` type resources. To do so, click into the `Filter for any field...` search box and type in `Storage account`.![azure-filter-storage-accounts](./images/azure-filter-storage-accounts.png)
+1. Click into the PHI storage account, which is where all Protected Health Information is stored outside of the FHIR server. The precise name of the storage bucket will have the form `phdi{environment}phi{clientId}`, e.g., `phdidevphi1667849158`.![azure-select-phi-bucket](./images/azure-select-phi-bucket.png)
+1. Go to the left sidebar and under the `Data storage` header, click `Containers`. ![azure-containers](./images/azure-containers.png)
+1. Click into the `source-data` container.![azure-select-source-data-container](./images/azure-select-source-data-container.png)
```

Comment:
nit: This screenshot doesn't have the red highlight like most of the other screenshots do

Reply:
ty for catching this! updated a few screenshots to include the red highlight!